src/main/java/com/epam/ms/handler/EventHandler.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.ms.handler;
+
+import com.epam.ms.repository.domain.Event;
+
+public interface EventHandler {
+    public abstract void handle(Event event);
```

Comment:
if this is an interface then just `void handle(Event event);`
it's public by default
Also, good to have some documentation above the interface itself

Reply:
Thank you, done🙈